### DISCUSSION TITLE
Welcome to faber Discussions!

### DISCUSSION THREAD

**Discussion by stefanseefeld** [stefanseefeld/faber#25] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PGDataHome** [comment]
Hi,

I'm using 
"Python 3.12.2 (main, Mar 29 2024, 13:10:56) [GCC 13.2.1 20240326 releases/gcc-13.2.0-888-g66090b8f43] on linux" 
on ClearLinux

I'a trying to rebuild boost-python because of this error message,
 "SystemError: type Boost.Python.enum has the Py_TPFLAGS_HAVE_GC flag but has no traverse function"*
but this boost module requires faber, so trying to install this faber, first last dev version 0.6, but on error tried release 0.4, and the same occurs, i get this

 "s/boost/faber-release-0.4.2/versioneer.py", line 342, in get_config_from_root
    parser = configparser.SafeConfigParser()
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'configparser' has no attribute 'SafeConfigParser'. Did you mean: 'RawConfigParser'?
"
